src/Learning/KDDomainKnowledge/CMakeLists.txt

```diff
@@ -1,7 +1,15 @@
 file(GLOB cppfiles ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
 add_library(KDDomainKnowledge STATIC "${cppfiles}")
 
+if(GENERATE_VIEWS)
+  target_sources(KDDomainKnowledge PRIVATE KDTextFeatureSpec.dd.log)
+endif()
+
 set_khiops_options(KDDomainKnowledge)
 
 target_include_directories(KDDomainKnowledge PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 target_link_libraries(KDDomainKnowledge PUBLIC KWDataPreparation)
+
+generate_gui(KDTextFeatureSpec "Text feature parameters" ../KWUserInterface/KDTextFeatureSpec.dd
+             KDTextFeatureSpec.dd.log -noview)
```

Comment:
`#cmake: off` ?

Reply:
No interest for me to add to line of comment for one line that is readable